src/tiled/propertybrowser.cpp

```diff
@@ -753,9 +754,9 @@ void PropertyBrowser::addTilesetProperties()
         auto property = addProperty(FileNameProperty, filePathTypeId(), tr("Filename"), groupProperty);
 
         QString filter = QCoreApplication::translate("MainWindow", "All Files (*)");
-        FormatHelper<TilesetFormat> helper(FileFormat::Read, filter);
+//        FormatHelper<TilesetFormat> helper(FileFormat::Read, filter);
 
-        property->setAttribute(QStringLiteral("filter"), helper.filter());
+//        property->setAttribute(QStringLiteral("filter"), helper.filter());
```

Comment:
What's happening here?

Reply:
Fixed

I was having compile issues so I suspect I commented this and hadn't put it back yet.